
From: Dave Jones <davej@redhat.com>

arch/i386/mach-generic/summit.c: In function `send_IPI_all':
include/asm/mach-summit/mach_ipi.h:4: sorry, unimplemented: inlining failed in call to 'send_IPI_mask_sequence': function body not available
arch/i386/mach-generic/summit.c:8: sorry, unimplemented: called from here
make[1]: *** [arch/i386/mach-generic/summit.o] Error 1
make: *** [arch/i386/mach-generic] Error 2

arch/i386/mach-generic/bigsmp.c: In function `send_IPI_all':
include/asm/mach-bigsmp/mach_ipi.h:4: sorry, unimplemented: inlining failed in call to 'send_IPI_mask_sequence': function body not available
arch/i386/mach-generic/bigsmp.c:8: sorry, unimplemented: called from here
make[1]: *** [arch/i386/mach-generic/bigsmp.o] Error 1
make: *** [arch/i386/mach-generic] Error 2

arch/i386/mach-generic/es7000.c: In function `send_IPI_all':
include/asm/mach-es7000/mach_ipi.h:4: sorry, unimplemented: inlining failed in call to 'send_IPI_mask_sequence': function body not available
arch/i386/mach-generic/es7000.c:8: sorry, unimplemented: called from here
make[1]: *** [arch/i386/mach-generic/es7000.o] Error 1
make: *** [arch/i386/mach-generic] Error 2


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-x86_64-akpm/include/asm-i386/mach-bigsmp/mach_ipi.h |    2 +-
 25-x86_64-akpm/include/asm-i386/mach-es7000/mach_ipi.h |    2 +-
 25-x86_64-akpm/include/asm-i386/mach-summit/mach_ipi.h |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff -puN include/asm-i386/mach-bigsmp/mach_ipi.h~267-rc3-mm2-inlining-failures include/asm-i386/mach-bigsmp/mach_ipi.h
--- 25-x86_64/include/asm-i386/mach-bigsmp/mach_ipi.h~267-rc3-mm2-inlining-failures	Mon Aug  9 17:29:47 2004
+++ 25-x86_64-akpm/include/asm-i386/mach-bigsmp/mach_ipi.h	Mon Aug  9 17:29:47 2004
@@ -1,7 +1,7 @@
 #ifndef __ASM_MACH_IPI_H
 #define __ASM_MACH_IPI_H
 
-inline void send_IPI_mask_sequence(cpumask_t mask, int vector);
+void send_IPI_mask_sequence(cpumask_t mask, int vector);
 
 static inline void send_IPI_mask(cpumask_t mask, int vector)
 {
diff -puN include/asm-i386/mach-es7000/mach_ipi.h~267-rc3-mm2-inlining-failures include/asm-i386/mach-es7000/mach_ipi.h
--- 25-x86_64/include/asm-i386/mach-es7000/mach_ipi.h~267-rc3-mm2-inlining-failures	Mon Aug  9 17:29:47 2004
+++ 25-x86_64-akpm/include/asm-i386/mach-es7000/mach_ipi.h	Mon Aug  9 17:29:47 2004
@@ -1,7 +1,7 @@
 #ifndef __ASM_MACH_IPI_H
 #define __ASM_MACH_IPI_H
 
-inline void send_IPI_mask_sequence(cpumask_t mask, int vector);
+void send_IPI_mask_sequence(cpumask_t mask, int vector);
 
 static inline void send_IPI_mask(cpumask_t mask, int vector)
 {
diff -puN include/asm-i386/mach-summit/mach_ipi.h~267-rc3-mm2-inlining-failures include/asm-i386/mach-summit/mach_ipi.h
--- 25-x86_64/include/asm-i386/mach-summit/mach_ipi.h~267-rc3-mm2-inlining-failures	Mon Aug  9 17:29:47 2004
+++ 25-x86_64-akpm/include/asm-i386/mach-summit/mach_ipi.h	Mon Aug  9 17:29:47 2004
@@ -1,7 +1,7 @@
 #ifndef __ASM_MACH_IPI_H
 #define __ASM_MACH_IPI_H
 
-inline void send_IPI_mask_sequence(cpumask_t mask, int vector);
+void send_IPI_mask_sequence(cpumask_t mask, int vector);
 
 static inline void send_IPI_mask(cpumask_t mask, int vector)
 {
_
